src/UsersModel.cpp

```diff
@@ -59,8 +79,7 @@ UsersModel::data(const QModelIndex &index, int role) const
         case CompletionModel::SearchRole2:
```

Comment:
The line above this should default to returning the mxid, if the displayname is not there. Otherwise some things have no name in some cases. Wasn't an issue before, but with your pills it is.

Reply:
I see the issue, but I'd rather fix that in the QML. Returning an mxid as a display name seems wrong because they are not the same. Also, showing a displayname only when a profile is available is a good indicator that the mxid was entered without typo. Checking whether mxid and displayname are the same is less intuitive and also slower.

Reply:
Yeah, I guess that works too.